Commands/restrictedWords.js

```diff
@@ -38,14 +38,34 @@ const words = [
     'whitehatjunior'
 ]
 
+const curseWords = [
+    'fuck',
+    'shit',
+    'bitch',
+    'asshole',
+    'ass',
+    'kys',
+    'kill your self',
+    'bastard',
+    'dick'
+]
+
 module.exports = (client, callback) => {
     client.on('message', message => {
-        for (var i=0; i < words.length; i++) {
+        for (let i=0; i < words.length; i++) {
             if(message.content.toLowerCase().includes(words[i])) {
                 message.reply('Please do not talk about other edTech startups here 🚫. If you think I did a mistake dont worry I am still under development, tag Community Manager and report this 🏷️. ');
                 message.author.send(warnEmbed);
                 message.delete();
             }
         }
+
+        for (let i = 0; i < curseWords.length; i++) {
+            if(message.content.toLowerCase().includes(curseWords[i])) {
+                message.reply('Please do not use profane language 🚫. If you think I did a mistake dont worry I am still under development, tag Community Manager and report this 🏷️.');
+                message.author.send(warnEmbed);
+                message.delete();
```

Comment:
Should this be there. What do you think?

Reply:
Hmmm. I've seen that this is included for tech startups so I thought I would include one for curse words too. Maybe we could remove it and make it only send private message. I think that would be better yea.

Reply:
Private and message under their message should  be good